packages/component/src/hooks/useActivities.js

```diff
@@ -0,0 +1,5 @@
+import { useSelector } from '../WebChatReduxContext';
+
+export default function useActivities() {
```

Comment:
Are you supposed to return values in an array when making custom hooks?

Reply:
Yap. This pattern is adopted from `useState()`, which return `[value: string, setter: (nextValue: string) => void]`.

For example, for settable properties, e.g. send box value, `useSendBox()` will return `['current text in send box', (nextValue) => void]`.

But for read-only properties, to maintain consistency on the signature, `useLanguage()` will return `['en-US']`. This also help feature detection, i.e. the developer can know if the property can be set or not by looking at the 2nd item of the array.